src/crypto/secp256k1.ts

```diff
@@ -1,20 +1,27 @@
 import ecc from "secp256k1";
-import { randomBytes } from "crypto";
+import {randomBytes} from "crypto";
+import {Bip39, EnglishMnemonic, Slip10, Slip10Curve, Slip10RawIndex} from "@cosmjs/crypto"
```

Comment:
```suggestion
import { randomBytes } from "crypto";
import { Bip39, EnglishMnemonic, Slip10, Slip10Curve, Slip10RawIndex } from "@cosmjs/crypto"
```
Also, please insert spaces inside brackets. 

Javascript/Typescripts sucks on this formatting. VSCode and Jetbrains have their own way to configure this. To enhance this, I will find some 'lint' practices.

Reply:
I fixed it.
This also changed the formatter in Intellij.